rest_docs/rest-catalog-open-api.yaml

```diff
@@ -564,12 +648,21 @@ components:
           example: '{ "owner": "Hank Bendickson" }'
           default: '{ }'
 
-    Namespace:
-      description: Reference to one or more levels of a namespace
-      type: array
-      items:
-        type: string
-      example: [ "accounting", "tax" ]
+    UpdateNamespacePropertiesRequest:
```

Comment:
Moved this from elsewhere. Everything else is additions.

Reply:
It's kind of weird there's not a dedicated `requests` section, huh?

Eventually, if we wanted to, we could place things into separate files. But then we'd need something to bundle them all up together so the trade-off might not be worth it.

Reply:
You can have a section of `requestBodies`. I considered refactoring to use that, but it is a larger change.